OpenRA.Game/Server/Server.cs

```diff
@@ -737,12 +737,17 @@ void HandleSyncOrder(int frame, byte[] packet)
 
 		public void DispatchOrdersToClients(Connection conn, int frame, byte[] data)
 		{
-			var from = conn?.PlayerIndex ?? 0;
+			var from = conn.PlayerIndex;
```

Comment:
Good thing I tested before merging! This looked safe to me, but it turns out that we do still have at least one case where `conn` is null - `SendMessage` below defaults to it in most cases. Not sure if you want to revert this line or change `SendMessage` to use `DispatchServerOrdersToClients`?

Reply:
LGTM and testing revealed no further obvious problems after reverting this line.